basic_commands/seeking.md

```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
 
 
 In the visual mode you can press `u` (undo) or `U` (redo) inside the seek history to return back to previous or forward to the next location.
+
+## Open file
+
+As test file lets use some simple hello_world.c compiled in Linux ELF format.
+After we compiled it lets open it with radare2
+
+    r2 ./hello_world
+
+Now we have command prompt
+
+    [0x00400410]>
+
+Now we are ready to go deeper. 
+
+## Seeking at any position
+
+Seek to address 0x0
+
+    [0x00400410]> s 0x0
+    [0x00000000]>
+
+Print current address 
+
+    [0x00000000]> s
+    0x0
+    [0x00000000]>
+
+Seek N positions forward
+
+    [0x00000000]> s+ 128
+    [0x00000080]>
+
+Undo/Redo seek. Lets see with positions we where seeking
+
+    [0x00400410] [0x00000000] [0x00000080]
```

Comment:
The seek history should be listed with the s! Command or the us .. I gues we need to fill an issue for that but i remember there was a way to do that but incant find it from the phone now.